package.json

```diff
@@ -87,7 +86,7 @@
     "**/react-syntax-highlighter": "^15.3.1",
     "**/react-syntax-highlighter/**/highlight.js": "^10.4.1",
     "**/request": "^2.88.2",
-    "**/trim": "0.0.3",
+    "**/trim": "1.0.1",
```

Comment:
__note:__ [really, nothing to worry about](https://github.com/Trott/trim/compare/v0.0.3...v1.0.1), just moved to a proper semver.